### PR TITLE
DB-10407 Add hdp3.1.5 platform

### PR DESCRIPTION
**Pull request by bklo94**

### CODE REVIEW

**Review by msirek — APPROVED** [review]
The changes look good.

**Review by OlegMazurov — CHANGES_REQUESTED** [review]
_HConfigurationTest_ unit test fails due to _guava_ version mismatch.
hadoop-common-3.1.1.3.1.5.124-1.jar requires

void checkArgument(boolean, java.lang.String, java.lang.Object);

which is available in guava-28.2-jre.jar, provided by HDP 3.1.5,  but not guava-12.0.1.jar, provided by Splice at test time.

File: hbase_sql/pom.xml [hunk 1]
```diff
@@ -192,6 +192,18 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (OlegMazurov) [inline hbase_sql/pom.xml:195]: Redundant. Fix _org.apache.hive:hive-jdbc_

File: splice_spark/pom.xml [hunk 2]
```diff
@@ -132,6 +132,17 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (OlegMazurov) [inline splice_spark/pom.xml:135]: Redundant. Fix _org.apache.hive:hive-jdbc_ in _hbase_sql_.

File: splice_spark2/pom.xml [hunk 3]
```diff
@@ -79,6 +79,17 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (OlegMazurov) [inline splice_spark2/pom.xml:82]: Redundant. Fix _org.apache.hive:hive-jdbc_ in _hbase_sql_.

**Review by OlegMazurov — CHANGES_REQUESTED** [review]
_HConfigurationTest_ unit test fails due to _guava_ version mismatch.
hadoop-common-3.1.1.3.1.5.124-1.jar requires

void checkArgument(boolean, java.lang.String, java.lang.Object);

which is available in guava-28.2-jre.jar, provided by HDP 3.1.5,  but not guava-12.0.1.jar, provided by Splice at test time.

**Comment by bklo94** [comment]
jenkins please test branch @hdp3.1.5

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1706/

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1710/

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1712/

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1713/

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1720/

**Comment by bklo94** [comment]
jenkins please test branch @hdp3.1.5,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1724/

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1723/

**Comment by jyuanca** [comment]
jenkins please test branch @hdp3.1.5

**Review by OlegMazurov — APPROVED** [review]

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**